Assets/HoloToolkit-Examples/UX/Scripts/Controls/SliderGestureControl.cs

```diff
@@ -83,6 +85,20 @@ protected override void Awake()
         {
             base.Awake();
 
+            if (MinSliderValue >= MaxSliderValue)
+            {
+                Debug.LogError("Your SliderGestureControl has a min value that's greater than or equal to its max value.");
+                Destroy(this);
```

Comment:
Why don't we just disable it instead of destroying it?

Reply:
Ahh, good question.
Disabling just the script didn't do anything, since it was driven through events. I updated this line to disable the entire GameObject, which prevents events being directed via focus.

Reply:
Okay so, the way this script is currently written, there's a GestureInteractive component that drives events into the SliderGestureControl. SliderGestureControl doesn't know about the GestureInteractive, so I'd have to search for it and disable both.

You were right about `enabled = false;`, if this was written as a single script responding to the input system's events (it feels like it probably could be, but that's outside the scope of this PR/issue).